libcontainer/configs/validate/validator.go

```diff
@@ -200,19 +200,16 @@ func (v *ConfigValidator) intelrdt(config *configs.Config) error {
 			return errors.New("intelRdt is specified in config, but Intel RDT is not supported or enabled")
 		}
 
+		if config.IntelRdt.ClosID == "." || config.IntelRdt.ClosID == ".." || strings.ContainsRune(config.IntelRdt.ClosID, '/') {
```

Comment:
I think `strings.ContainsRune` is overkill here, as we don't care about unicode, and `strings.Contains` should be sufficient (and probably faster).

Reply:
Hmm, good point! I was thinking that `ContainsRune` would be better than substring matching but now that I look at the source code of the strings package that's not the case. Changed to `strings.Contains()`